lib/netbuf/netbuf.h

```diff
@@ -58,8 +58,8 @@ void netbuf_read_free(struct netbuf_read *);
  * Create and return a buffered writer attached to socket ${s}.  The caller
  * is responsible for ensuring that no attempts are made to write to said
  * socket except via the returned writer until netbuf_write_free() is called.
- * to destroy the writer.  If a write fails, ${fail_callback} will be invoked
```

Comment:
It would also be grammatical to merely delete the previous period.  However, we normally don't mention "to destroy the foo" when mentioning "bar_free(), so I'm guessing that this is what you intended in 2011.